features/functions/create_data.rb

```diff
@@ -18,10 +19,21 @@ def generate_new_borrower_with_middle
   }
 end
 
+# rubocop:disable all
 def create_deed_hash(title_number = generate_title_number,
-                     new_borrower1 = generate_new_borrower_no_middle)
+                     new_borrower1 = generate_new_borrower_no_middle,
+                     gender = generate_gender_male,
+                     address = generate_borrower_address,
+                     dob = generate_dob,
+                     phone_number = generate_phone_number
+                    )
   {
     title_number: title_number,
-    borrowers: [new_borrower1]
+    borrowers: [new_borrower1],
+    gender: [gender],
+    address: [address],
+    dob: [dob],
+    phone_number: [phone_number]
   }
 end
+# rubocop:enable all
```

Comment:
Thought I would put something in here about the fact that the above has failed rubocop because we are passing in too many parameters as part of the method call, but we are agreeing to proceed anyway as this method of creating data is going to be replaced next sprint.